code/drasil-docLang/Drasil/ExtractDocDesc.hs

```diff
@@ -43,26 +43,26 @@ secConPlate mCon mSec = preorderFold $ purePlate {
 }
 
 -- | Creates a section plate for expressions.
-exprPlate :: DLPlate (Constant [Expr])
+exprPlate :: DLPlate (Constant [DisplayExpr])
 exprPlate = sentencePlate (concatMap sentToExp) `appendPlate` secConPlate (concatMap egetCon')
   (concatMap egetSec) `appendPlate` (preorderFold $ purePlate {
   scsSub = Constant <$> \case
-    (TMs _ _ t) -> let r = concatMap (\x -> x ^. invariants ++
-                           defExp (x ^. defined_quant ++ x ^. defined_fun) ++
-                           r (x ^. valid_context)) in r t
-    (DDs _ _ d _) -> map sy d ++ defExp d
-    (GDs _ _ g _) -> expRel g
-    (IMs _ _ i _) -> expRel i
+    (TMs _ _ t)   -> goTM t
+    (DDs _ _ d _) -> go d -- ++ map (toDispExpr . sy) d ++ map (toDispExpr . (^. defnExpr)) d -- TODO: this *was* a very different definition in comparison to those around it...
```

Comment:
The commented out method is a 1-1 way of translating the existing code into this new DisplayExpr language, but it seems to make no difference between this way or the way the current code works either.